worker/jobTypes/productionDeployJob.js

```diff
@@ -27,8 +27,11 @@ async function verifyBranchConfiguredForPublish (currentJob) {
   const repoContent = await workerUtils.getRepoPublishedBranches(repoObject);
   if (repoContent && repoContent.status === 'success') {
     const publishedBranches = repoContent.content.git.branches.published;
+    //if this is stable branch AND [its the primary alias OR no aliases exist], then we want to use this build's manifest for global search
+    currentJob.payload["stableBranch"] = (repoContent.content.version.stable === currentJob.payload.branchName && (currentJob.payload.primaryAlias || ! currentJob.payload.aliased) )   ? '-g' : "";
```

Comment:
[nit] We probably don't want a space here
```suggestion
    currentJob.payload["stableBranch"] = (repoContent.content.version.stable === currentJob.payload.branchName && (currentJob.payload.primaryAlias || !currentJob.payload.aliased) )   ? '-g' : "";
```